packages/components/icon/src/Icon.tsx

```diff
@@ -0,0 +1,32 @@
+import * as VisuallyHidden from '@radix-ui/react-visually-hidden'
+import React, { PropsWithChildren } from 'react'
+
+import { iconVariants, IconVariantsProps } from './Icon.variants'
+
+export interface IconProps
```

Comment:
Maybe we should ensure (with TS) that only one child can be passed to this component, like this:
```tsx
export interface IconProps extends IconVariantsProps, React.SVGProps<SVGElement> {
  /**
   * The accessible label for the icon. This label will be visually hidden but announced to screen
   * reader users, similar to `alt` text for `img` tags.
   */
  label?: string
  children: React.ReactElement
}
```

usage:
```tsx
<Icon> // 
  <svg></svg>
  <svg></svg>
</Icon>
```
⬆️ , you'll get a TS error
```ts
This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | JSXElementConstructor<any>>', but multiple children were provided.
```